modules/framework-cats/shared/src/test/scala/SnapshotExpectations.scala

```diff
@@ -0,0 +1,35 @@
+package weaver
+package framework
+package test
+
+import cats.effect.IO
+
+import snapshot4s.*
+
+object SnapshotExpectations extends SnapshotAssertions[IO[Expectations]] {
```

Comment:
For context, should we add a comment linking to https://siriusxm.github.io/snapshot4s/contributing/supporting-a-test-framework#integrating-your-own-test-framework?